src/main/java/openfasttrack/report/PlainTextReport.java

```diff
@@ -140,23 +147,10 @@ private void renderFailureIds(final PrintStream report)
     private void renderFailureDetails(final PrintStream report)
     {
         this.trace.getUncoveredItems().stream() //
-                .sorted((item, other) -> item.getId().compareTo(other.getId())) //
```

Comment:
Fixed.
Actually an empty "sorted()" is enough if the streamed object implements "Comparable".